internal/appsec/waf.go

```diff
@@ -195,8 +219,18 @@ func newGRPCWAFEventListener(handle *waf.Handle, _ []string, timeout time.Durati
 			mu.Unlock()
 		}))
 		op.On(grpcsec.OnHandlerOperationFinish(func(op *grpcsec.HandlerOperation, _ grpcsec.HandlerOperationRes) {
+			op.AddMetric("_dd.appsec.waf.duration", float64(wafRunDuration/1e3))
+			op.AddMetric("_dd.appsec.waf.duration_ext", float64(wafBindingsRunDuration/1e3))
+
+			metricsOnce.Do(func() {
+				rInfo := handle.RulesetInfo()
+				op.AddMetric("_dd.appsec.event_rules.version", rInfo.Version)
```

Comment:
Copy/paste police 👮‍♂️: can you please have the metric key names as global const so they can be shared between the two callbacks and we avoid mistakes :-)